geom/type_polygon.go

```diff
@@ -2,6 +2,7 @@ package geom
 
 import (
 	"database/sql/driver"
+	"fmt"
```

Comment:
(commenting here since the file I would comment on for this change isn't in the diff)

The other thing that would be good to add is `Summary() string` and `String() string` on the `Geometry` type. This will unfortunately involve one of those nasty 7-case switch statements (at least until https://github.com/peterstace/simplefeatures/issues/399 is looked at).

This will actually make the testing a little bit easier, since you don't have to call the `Summary` methods directly on each concrete geometry. You can use something like:

```
wkt := "POLYGON((0 0,0 1,1 0,0 0))"
g := geomFromWKT(t, wkt)
got := g.Summary()
// check got vs want
```

Because you're operating on the generic `Geometry` type at that point, you can actually do the Summary tests cases for all 7 types all in the same unit test (since your test cases would just be `string inputWKT` and `string wantSummary`). It may also make sense to put these tests into `geom/attr_test.go`, which contains the other tests that calculate geometry "attributes" (in a sense, the summary string is a calculated "attribute" of a geometry).

Reply:
i like the suggestion; i noticed I was repeating a very similar pattern on each geometry's test after adopting the WKT approach.